options.c

```diff
@@ -125,7 +128,9 @@ int attrs[NR_ATTRS] = {
 	A_NORMAL,
 	A_NORMAL,
 	A_NORMAL,
-	A_NORMAL
+	A_NORMAL,
```

Comment:
While moving new entries to the bottom, I realized that one was acutally missing here.